features/url-formattering.feature

```diff
@@ -0,0 +1,127 @@
+#language: nl
+
+Functionaliteit: url formattering
+
+    Als consumer van de Haal Centraal APIs
+    Wil ik kunnen aangeven hoe de urls in een response moeten worden geformatteerd
+    Zodat ik de urls op mijn gewenste manier kan verwerken.
+
+    Als provider van een Haal Centraal API
+    Wil ik [URI Template](https://datatracker.ietf.org/doc/html/rfc6570) gebruiken om URIs te beschrijven
+    Zodat een verzameling van gelijksoortige urls met één URI template kan worden gerepresenteerd.
+
+    De Haal Centraal APIs gebruiken [server-driven negotiation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Content_negotiation)
```

Comment:
Je schrijft dit algemeen voor Haal Centraal. Maar in eerste instantie geldt dit denk ik alleen voor de BRP API?

Reply:
Ja, als we dit een goede oplossing vinden, dan is het wel handig als alle Haal Centraal APIs het implementeren.

Reply:
We doen dit juist om het voor afnemers mogelijk te maken (al) onze API's compliant te laten opereren met de eigen (API Gateway) policies.